src/app/services/createRapid.js

```diff
@@ -0,0 +1,37 @@
+import http from "@/app/http"
```

Comment:
should these functions be separate files? Or should we have like `app/services/rapids.js` that exports each function (createRapid, deleteRapid, etc.)

Reply:
i.e. we would organize by "model" (even though the graphQL endpoints are all kind of their own thing)

Reply:
ahh, yeah. Good point. That'd definitely clean things up even more. I'm all for it. Is that a show stopper for this PR?

Reply:
no, the only reason I would be inclined to include it in this one is so we don't forget to do it? But it's fine if you just want to get this merged